src/Diagnostics.ModelsAndUtils/Models/OperationContext.cs

```diff
@@ -48,7 +48,12 @@ public class OperationContext<TResource> where TResource : IResource
         /// </summary>
         public string TimeGrain { get; private set; }
 
-        public OperationContext(TResource resource, string startTimeStr, string endTimeStr, bool isInternalCall, string requestId, string timeGrain = "5", SupportTopic supportTopic = null)
+        /// <summary>
+        /// ExecuteForm object
+        /// </summary>
+        public FormContext Form { get; private set; }
```

Comment:
If you can add multiple forms in one detector, would there be multiple form context objects that would need to be passed?

Reply:
FormContext contains data of the form that is executing. A user can click only one button at at time. So at any point of time, there will be only one FormContext object in a detector.